src/main/java/se/kth/sda/skeleton/posts/exeption/ResourceNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package se.kth.sda.skeleton.posts.exeption;
```

Comment:
just a minor error on the package name : ex"c"eption , missing letter c

Reply:
Renamed the package.